### PR TITLE
Use Humble branch reference and correct branch for CI

### PR DESCRIPTION
**Pull request by fmauch**
We need to branch out for humble, this sets up the main branch accordingly regarding README and CI

### CODE REVIEW

File: .github/workflows/humble-semi-binary-main.yml [hunk 1]
```diff
@@ -18,5 +18,5 @@ jobs:
     with:
       ros_distro: humble
       ros_repo: main
-      ref_for_scheduled_build: ros2
-      upstream_workspace: Universal_Robots_ROS2_Gazebo_Simulation.humble.repos
+      ref_for_scheduled_build: humble
+      upstream_workspace: Universal_Robots_HUMBLE_Gazebo_Simulation.humble.repos
```
Comment (RobertWilbrandt) [inline .github/workflows/humble-semi-binary-main.yml]: Is this intentional?

Comment (fmauch) [reply .github/workflows/humble-semi-binary-main.yml]: grml...

**Comment by fmauch** [comment]
Failing semi binary seems to be MoveIt build problems on rolling.

**Review by RobertWilbrandt — APPROVED** [review]
Looks good now, thanks

**Pull request merged**